push/parser.go

```diff
@@ -148,8 +172,11 @@ func (p *NotificationParserImpl) parseUpdate(data *genericData, nested *genericM
 	}
 
 	base := BaseUpdate{
-		BaseMessage:  BaseMessage{timestamp: data.Timestamp, channel: data.Channel},
-		changeNumber: nested.ChangeNumber,
+		BaseMessage:          BaseMessage{timestamp: data.Timestamp, channel: data.Channel},
+		changeNumber:         nested.ChangeNumber,
+		previousChangeNumber: nested.PreviousChangeNumber,
```

Comment:
`previousChangeNumber` `compressType` and the data should be instantiated only in `case UpdateTypeSplitChange:`

Reply:
I moved these properties to SplitChangeUpdate.